articles/server-platforms/apache.md

```diff
@@ -17,34 +17,28 @@ snippets:
 
 ### 1. Install and enable `mod_auth_openidc` module
 
-${snippet(meta.snippets.dependencies)}
 First, you need to install the `mod_auth_openidc` module for Apache.
 
 You can get the binaries from [Github](https://github.com/pingidentity/mod_auth_openidc/releases) and install them for your OS. If your OS isn't compatible with any of the binaries, you can still [build it from source](https://github.com/pingidentity/mod_auth_openidc/blob/master/INSTALL)
 
-Once you've installed it, you just need to enable it for Apache
+Once you've installed it, you just need to enable it for Apache (If you are using Windows, you can use [this](https://github.com/enderandpeter/win-a2enmod#installation) to get `a2enmod` working on your system)
 
 ${snippet(meta.snippets.dependencies)}
 
 ### 2. Configure the module with your Auth0 Account information
 
-Now, you should get a new configuration file under the `/etc/apache2/mods-available` folder, where Apache modules are normally installed.
+Now, you should get a new configuration file under the `/apache2/mods-available` folder, where Apache modules are normally installed.
```

Comment:
But if they aren't using Windows this will be confusing, no? We should probably explain that the path can be `/etc/` if they are on a different platform.